lib/backbeat/Metrics.js

```diff
@@ -410,6 +410,50 @@ class Metrics {
         });
     }
 
+    /**
+     * Get pending replication stats by ops count and size in bytes
+     * @param {object} details - route details from lib/backbeat/routes.js
+     * @param {function} cb - callback(error, data)
+     * @return {undefined}
+     */
+    getPending(details, cb) {
+        const { site, dataPoints } = details;
+        // Query only a single given site.
```

Comment:
Are there plans to do a 'getPending' for all sites?

Reply:
Yes, I'm working on including it in the response of https://github.com/scality/backbeat/blob/development/8.0/docs/metrics.md#_backbeatapimetricscrrlocation-name